esp/src/eclwatch/DynamicESDLMethodWidget.js

```diff
@@ -211,7 +211,7 @@ define([
             }).then(function (response) {
                 if (lang.exists("ConfigureESDLBindingMethodResponse.status", response)) {
                     if (response.ConfigureESDLBindingMethodResponse.status.Code === 0) {
-                        context.store.remove(selection[idx].__hpcc_id);
+                        //  MV Is this needed (idx is invalid)?:  context.store.remove(selection[idx].__hpcc_id);
```

Comment:
Sorry been working on a few things, I have not forgotten about this. Let me test with some DESDL examples and respond here.